lib/api/pokedex_api_spec/api_service.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:pokedex/api/pokedex_api_spec/api_client.dart';
+import 'package:pokedex/api/pokedex_api_spec/handler/pokemon_handler.dart';
+
+class ApiService {
```

Comment:
for the purpose of guaging your learning here, may I ask you what kind of pattern is this and why do you think this is a good move? 🙂

Reply:
this is singleton pattern it lets you have only 1 instance of this object in the whole app because it would be easier to call this class over and over again